sql/proposals/proposals.go

```diff
@@ -129,9 +149,17 @@ func GetBlob(db sql.Executor, id []byte) (proposal []byte, err error) {
 	return proposal, err
 }
 
+func toTransactionIDs(txIDs []types.TransactionID) TransactionIDs {
+	if txIDs == nil {
+		return nil
+	}
+	return TransactionIDs(txIDs)
+}
```

Comment:
whats wrong with TransactionIDs(nil) ?

Reply:
removed it as per https://github.com/spacemeshos/go-spacemesh/pull/3432#discussion_r944148370